setup.py

```diff
@@ -93,6 +93,7 @@ def run_tests(self):
     cmdclass={'test': PyTest},
     tests_require=test_requirements,
     extras_require={
+        'cchardet:(python_version == "2.7" or python_version >= "3.4")': ['cchardet>=2.1.0'],
```

Comment:
If we're enforcing (in `__init__.py`) that `cchardet` be between `2.1.0` and `2.2.0` then that should be written here.